fxevent/zap.go

```diff
@@ -199,3 +200,10 @@ func moduleField(name string) zap.Field {
 	}
 	return zap.String("module", name)
 }
+
+func trueBool(name string, b bool) zap.Field {
```

Comment:
nit: I think the name should hint at skipping behavior e.g. trueOrSkip or maybeBool. Something that might flag a someone skimming the code even more that there is something special.